schema.sql

```diff
@@ -6,5 +6,6 @@ id int GENERATED ALWAYS AS IDENTITY,
  date_of_birth date,
 escape_attempts int,
 neutered boolean,
-weight_kg decimal
+weight_kg decimal,
+ALTER TABLE animals ADD species varchar(255);
```

Comment:
- [ ] Kindly consider adding this `ALTER TABLE animals ADD species varchar(255);` outside of the block `create table animals ()` because adding `ALTER TABLE` inside `create table` is wrong. Your end code would look like the following:
```
create table animals (
    id int GENERATED ALWAYS AS IDENTITY,
     name varchar(250),
     date_of_birth date,
    escape_attempts int,
    neutered boolean,
    weight_kg decimal,
    species VARCHAR(50)
)
ALTER TABLE animals ADD species varchar(255);
```